File: 8KYU/altERnaTIng cAsE = ALTerNAtiNG CaSe/solution.cs
using System;
using System.Linq;
namespace Extensions
{
  public static class StringExt
  {
    public static string ToAlternatingCase (this string s)
      => string.Join("", s.ToCharArray()
          .Select(c =>char.IsUpper(c) ? char.ToLower(c) : char.ToUpper(c)));
    
  }
}

File: 8KYU/altERnaTIng cAsE = ALTerNAtiNG CaSe/solution.go
package kata

import (
  "unicode"
)

func ToAlternatingCase(str string) string {
  res := ""
  
  for _, c := range str {
    if unicode.IsUpper(c) {
      res += string(unicode.ToLower(c))
    } else {
      res += string(unicode.ToUpper(c))
    }
  }
  
  return res
}